### PR TITLE
code clean up. comments and unused code

### PR DESCRIPTION
**Pull request by ilanDoron**

### CODE REVIEW

File: contracts/sol5/utils/Utils4.sol [hunk 1]
```diff
@@ -6,7 +6,8 @@ import "../IERC20.sol";
 /**
  * @title Kyber utility file
  * mostly shared constants and rate calculation helpers
- * inherited by most of kyber contracts
+ * inherited by most of kyber contracts.
+ * previous utils implementations are for previous solidity version.
```
Comment (Anyhowclick) [inline contracts/sol5/utils/Utils4.sol:10]: versions

Comment (ilanDoron) [reply contracts/sol5/utils/Utils4.sol:10]: but before version 5 we have one version. (for our code)

File: contracts/sol6/mock/MaliciousKyberNetwork.sol [hunk 2]
```diff
@@ -111,23 +111,14 @@ contract MaliciousKyberNetwork is KyberNetwork {
...
-    /// @param destAddress Address to send tokens to
-    /// @return true if trade is successful
     function doReserveTrades(
         IERC20 src,
         uint256 amount,
         IERC20 dest,
         address payable destAddress,
         TradeData memory tData,
         uint256 expectedDestAmount
-    ) internal override returns (bool) {
```
Comment (manhlx3006) [inline contracts/sol6/mock/MaliciousKyberNetwork.sol:130]: Should remove amount here?

File: contracts/sol6/mock/MaliciousKyberNetwork2.sol [hunk 3]
```diff
@@ -1,30 +1,20 @@
...
+        myFeeWei = 10;
     }
 
-    /// @notice use token address ETH_TOKEN_ADDRESS for ether
-    /// @dev do one trade with a reserve
-    /// @param src Src token
-    /// @param amount amount of src tokens
-    /// @param dest   Destination token
-    /// @param destAddress Address to send tokens to
-    /// @return true if trade is successful
```
Comment (manhlx3006) [inline contracts/sol6/mock/MaliciousKyberNetwork2.sol:27]: Should remove amount here?

File: contracts/sol6/mock/MaliciousKyberNetwork.sol (hunk 2 above)
Comment (ilanDoron) [reply contracts/sol6/mock/MaliciousKyberNetwork.sol:130]: malicious network is used to send a bit less dest amount. add see the proxy can identify and revert.

so this function is doing it well. don't think we should re write it.

**Pull request merged**